tests/dataprep/test_dataprep_multimodal_redis_langchain.sh

```diff
@@ -256,6 +261,30 @@ function validate_microservice() {
         echo "[ $SERVICE_NAME ] Content is as expected."
     fi
 
+    # test v1/ingest_with_text with a PDF file
```

Comment:
There's a test below for the `get_files` endpoint. Could you add to that test to check that we see the PDF file in the output of that call?